The-Dragonss/TeleSoma
# add dashboard, signup and sign in, hero section

### Hey! I just made a pull request

### Description

This PR includes the implementation of the **Sign-In**, **Sign-Up**, **Dashboard**, and **Hero** sections for the application. These features provide a seamless user authentication experience and a new landing page layout.

### Type of Change

Please check the relevant options:


### How Has This Been Tested?

The changes have been tested across different environments to ensure compatibility. To view the updates:
1. Run the app.
2. Navigate to the **Landing Page** to view the new **Hero** section.
3. Access the **Sign-In** and **Sign-Up** pages from the navbar.
4. Once signed in, users are directed to the **Dashboard**.

### Screenshots (if applicable)

Please include any relevant screenshots of the **Sign-In**, **Sign-Up**, **Dashboard**, and **Hero** sections.

### Additional Notes

Feel free to check for any responsive design issues or UX improvements.

---

Thank you for your contribution!
